### PR TITLE
SYNR-1315

### PR DESCRIPTION
**Pull request by kimyen**

### CODE REVIEW

**Comment by kimyen** [comment]
Hi @karawoo & @kdaily 
If you have time, would you take a look at the NEWS.md file? How is the format of this file? Please feel free to comment/ fix my word choice.

File: NEWS.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
+# synapser 0.3
+
+## New features
+
+* New convenience function `synBuildTable` creates a Table Schema based on the given data, and returns a Table object that can be stored in Synapse using `synStore`.
+* New convenienve function `synMove` allows users to move entities to a different parent.
```
Comment (karawoo) [inline NEWS.md]: Typo here, should be "convenience"

File: NEWS.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+
+## Minor bug fixes and improvements
+
+* `synStore` now uses single threaded to avoid the C-stack limit error (SYNR-1323).
+* Creating Links now works for creating 50+ Links at a time (SYNR-1276).
+* Warnings from Python is no longer shown.
+* `synapser` namespace is no longer contains `PythonEmbedInR` and other R packages functions (SYNR-1274).
+* New NEWS.md provides the change log for each released version (SYNR-1315).
+* New [System Dependencies vignette](./articles/systemDependencies.html) explains what dependencies are required and how to installed them (SYNR-1328).
```
Comment (karawoo) [inline NEWS.md]: These relative path links don't work when clicked from GitHub, but I guess they will if this is rendered as a page on https://sage-bionetworks.github.io/synapser/

Comment (kimyen) [reply NEWS.md]: Yes, I will need to test out all of the link after this PR is merged.

**Comment by kimyen** [comment]
Thank you so much @kdaily & @karawoo !

File: NEWS.md [hunk 3]
```diff
@@ -0,0 +1,18 @@
+# synapser 0.3
+
+## New features
+
+* New convenience function `synBuildTable` creates a Table Schema based on the given data, and returns a Table object that can be stored in Synapse using `synStore`.
+* New convenience function `synMove` allows users to move entities to a different parent.
+
+## Minor bug fixes and improvements
+
+* `synStore` now uses a single thread to avoid the C stack limit error (SYNR-1323).
```
Comment (brucehoff) [inline NEWS.md]: I would replace the Jira IDs with links to the Jira issues.   If you do this, make sure the issues are viewable by anonymous Jira users.

**Pull request merged**